test-cases/longevity/longevity-large-partition-4days.yaml

```diff
@@ -2,8 +2,20 @@ test_duration: 6480
-stress_cmd: ["scylla-bench -workload=uniform -mode=read -replication-factor=3 -partition-count=10 -clustering-row-count=10000000 -clustering-row-size=5120 -rows-per-request=10 -concurrency=7 -max-rate=32000 -duration=5760m"]
+prepare_write_cmd:  ["scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=6 -clustering-row-count=10000000 -clustering-row-size=2048 -concurrency=300 -rows-per-request=2000 -timeout=180s -connection-count 200",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=6 -clustering-row-count=10000000 -partition-offset=6 -clustering-row-size=2048 -concurrency=300 -rows-per-request=2000 -timeout=180s -connection-count 200",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=10000000 -partition-offset=12 -clustering-row-size=2048 -concurrency=300 -rows-per-request=2000 -timeout=180s -connection-count 200",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=10000000 -partition-offset=19 -clustering-row-size=2048 -concurrency=300 -rows-per-request=2000 -timeout=180s -connection-count 200",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=10000000 -partition-offset=26 -clustering-row-size=2048 -concurrency=300 -rows-per-request=2000 -timeout=180s -connection-count 200",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=10000000 -partition-offset=33 -clustering-row-size=2048 -concurrency=300 -rows-per-request=2000 -timeout=180s -connection-count 200",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=10000000 -partition-offset=40 -clustering-row-size=2048 -concurrency=300 -rows-per-request=2000 -timeout=180s -connection-count 200",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=10000000 -partition-offset=47 -clustering-row-size=2048 -concurrency=300 -rows-per-request=2000 -timeout=180s -connection-count 200",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=6 -clustering-row-count=10000000 -partition-offset=54 -clustering-row-size=2048 -concurrency=300 -rows-per-request=2000 -timeout=180s -connection-count 200"
+]
+stress_cmd: ["scylla-bench -workload=uniform -mode=read -replication-factor=3 -partition-count=60 -clustering-row-count=10000000 -clustering-row-size=2048 -rows-per-request=2000 -timeout=180s -concurrency=10000 -max-rate=32000  -duration=5760m -connection-count 1000",
```

Comment:
@yarongilor are you sure about these values:
"-concurrency=10000 -max-rate=32000 " ?

What is the load during the stress (not the prepare stage)?

Reply:
If you have live monitor or test-id I can use to restore the monitor, it would be great.
If not, we can merge and test it as part of 4.6 and make some adjustments to it if needed.

Reply:
i don't remember how we got to this number of 10k, but it does make sence to change it to 1k.
we can also consider decreasing the connection count to 500. but perhaps, i'll try first rerunning it only changing the concurrency to 1K.

Reply:
> @yarongilor are you sure about these values: "-concurrency=10000 -max-rate=32000 " ?
> 
> What is the load during the stress (not the prepare stage)?

the read load was ~ 130K ops and 25% load on last updated run.